src/apps/frontend/components/otp/otp-input.tsx

```diff
@@ -4,7 +4,7 @@ import Input from '../input';
 
 type OTPInputProps = {
   disabled: boolean;
-  error: string;
+  error: string | false | undefined;
```

Comment:
Why would this be false?

This should be `error?: string` or `error: Nullable<string>`